### PR TITLE
Add github actions CI config

### PR DESCRIPTION
**Pull request by qishipengqsp**
Summary: add GitHub actions config to initialize CI
TODO:
- determine the CI trigger policy and modify the configuration accordingly
- add larger GitHub-hosted runners to the org to speed up CI
- confirm and optimize the CI scripts
- add the CI status badge to the README in the project

### CODE REVIEW

**Comment by qishipengqsp** [comment]
And consider the CD configuration, multi platform configuration

**Comment by qishipengqsp** [comment]
> configuration

Have discussed it. Do not config CD currently

**Comment by qishipengqsp** [comment]
> Summary: add GitHub actions config to initialize CI TODO:
> 
> * determine the CI trigger policy and modify the configuration accordingly
The current policy is ok.
> * add larger GitHub-hosted runners to the org to speed up CI
Larger runners is tested in Beta
> * confirm and optimize the CI scripts
Downloaded cov.py
> * add the CI status badge to the README in the project
added

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+name: ci
+
+on:
+  push:
+  pull_request:
+    branches:
+      - 'main'
+
+env:
+  TEST_TAG: tugraph/tugraph-ut:test
```
Comment (knightast) [inline .github/workflows/ci.yml:10]: tag标个版本号

**Pull request merged**